openstudiocore/src/openstudio_lib/HVACSystemsController.cpp

```diff
@@ -199,12 +200,15 @@ void HVACSystemsController::update()
     systemComboBox->clear();
 
     // Populate system combo box
-    std::vector<model::Loop> loops = m_model.getModelObjects<model::Loop>();
+    auto airloops = m_model.getModelObjects<model::AirLoopHVAC>();
```

Comment:
Stupid nitpick: getConcreteModelObjects would be faster, otherwise this looks great